docs/ci_example.md

```diff
@@ -0,0 +1,111 @@
+# Example 5: Incorporate fal into a CI/CD pipeline
+You can use fal as part of your CI/CD pipeline. In this example we use [Github Actions](https://github.com/features/actions).
+
+## Environment variables
+Environemt variables need to be provided as [repository secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) These will be different depending on which sources and outputs your project uses. In this example we use [BigQuery](https://cloud.google.com/bigquery/) both as a source and as an output, and so we need to provide BigQuery-specific environment variables:
+
+- `SERVICE_ACCOUNT_KEY`: contents of a `keyfile.json`. For more information see [here](https://docs.github.com/en/actions/security-guides/encrypted-secrets)
+- `GCLOUD_PROJECT`: your Google Cloud project ID.
+- `BQ_DATASET`: name of the BigQuery dataset
+
+If your fal scripts require environment variables, these should also be provided as repository secrets.
+
+## Setup dbt project
+
+### `profiles.yml`
+Since every dbt setup has it's own `profiles.yml`, usually in `~/.dbt` directory, there can be discrepancies between different setups. We therefore need a standard `profiles.yml` that will be used in the CI/CD workflow. Here's a profiles.yml that is specific to this example project:
+
+```yaml
+fal_dbt_examples:
+  target: dev
+  outputs:
+    dev:
+      type: bigquery
+      method: service-account
+      keyfile: "{{ env_var('KEYFILE_DIR') }}/keyfile.json"
+      project: "{{ env_var('GCLOUD_PROJECT') }}"
+      dataset: "{{ env_var('BQ_DATASET') }}"
+      threads: 1
+      timeout_seconds: 300
+      location: US
+      priority: interactive
+```
+
+As you can see, it uses [environment variables](#environment-variables) for some properties, as it's best to keep these secret.
+
+### `requirements.txt`
+All the packages that are necessary to run dbt and fal should be put in `requirements.txt`. This includes any packages that are used by user-defined python scripts. Here's an example of how `requirements.txt` can look like:
+
+```
+# Core dependencies
+dbt-core==0.21.0
+dbt-bigquery==0.21.0
+fal==0.1.26
+
+# Script dependencies
+slack_sdk
+datadog_api_client
+```
+
+## Action Workflow
+
+### Install dependencies
+The first step in our workflow is to setup python and install the dependencies from `requirements.txt`:
+
+```yaml
+- uses: actions/setup-python@v2
+  with:
+    python-version: '3.8.x'
+
+- name: Install dependencies
+  run: |
+    pip install --upgrade --upgrade-strategy eager -r requirements.txt
+```
+
+### Setup dbt profile
+Next, we setup `profiles.yml`:
```

Comment:
You could use `dbt --profile-dir` flag instead

Reply:
profile-dir is still going to be relative, so why not just put it in ~/.dbt

Reply:
Also, every dbt command that we would need to run in a workflow would require this flag. I think it's much easier to just write `dbt run` or `dbt seed`.

Reply:
I think it's just an opinion / preference thing. So either works!